### PR TITLE
ticket 4

### PR DESCRIPTION
**Pull request by rosiefhiggins**

### CODE REVIEW

File: src/Components/Reviews.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import { getReviews } from "../api"
+import { useEffect, useState } from 'react'
+
+const Reviews = () => {
+    const [reviewList, setReviews] = useState([])
+
+    useEffect(()=>{
+        getReviews()
```
Comment (leekli) [inline src/Components/Reviews.jsx:11]: Excellent, good to see the api call has been extracted

File: src/Components/Reviews.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        .then((reviews)=>{
+            setReviews(reviews)
+        })
+    }, [reviewList])
+    return (
```
Comment (leekli) [inline src/Components/Reviews.jsx:26]: Let's get an is loading design pattern in here

File: src/Components/Reviews.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <div>
+            <h2> All Reviews </h2>
+            <div className="reviews">
+            <ol>
+                {reviewList.map((review)=>{
+                    return (
+                        <li className="review" key={review.review_id}>
+                            <h3>{review.title}</h3>
+                            <br></br>
```
Comment (leekli) [inline src/Components/Reviews.jsx]: This looks good, I'd propose maybe adding the author name too? That would be a nice summary with just enough but not too much info on the article... showing the user the article title, author name and an image

File: src/api.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+const gamesApi=axios.create({
+    baseURL: "https://rosie-nc-games.onrender.com"
+});
+
+export const getReviews = () => {
+    return gamesApi.get("/api/reviews").then((res)=>{
+        return res.data.reviews
+    })
+}
```
Comment (leekli) [inline src/api.js]: Excellent, good to see the api call has been extracted

**Review by leekli — CHANGES_REQUESTED** [review]
Hey Rosie, well done so far! Good work.

The app is responsive and working in mobile mode. I've added a few comments for you to read.

The main request for change is: There is currently no indication to me as a user that the data is loading/being fetched, so if I was on a slow connection then there would be a blank screen for ages. Please could you implement an is loading design pattern into this page, and then submit a new pull request for review

**Comment by rosiefhiggins** [comment]
I've accidentally messed up my branches, so its got all my latest changes as well

**Review by leekli — APPROVED** [review]
Thanks for adding this Rosie.

Be careful about merging this as I can see your ticket 5 code is on here too, I'd probably resubmit this same branch once you've completed the ticket 5 code also to saves any issues?

**Pull request merged**